Extensions/go_to_dash.js

```diff
@@ -48,7 +48,7 @@ XKit.extensions.go_to_dash = new Object({
 			var blog = XKit.iframe.get_tumblelog();
 			go_back_html = html_pieces.join('dashboard/blog/' + blog + '/' + post_id);
 		} else {
-			var next_post_id = parseInt(post_id) + 1;
+			var next_post_id = BigInt(post_id) + 1;
```

Comment:
this is the one i'm most wary of. go_to_dash is widely used, and BigInt is only supported in firefox versions >= 68 (since July 2019). Suggestions welcome.

Reply:
i think the best thing here would be to feature-detect on the non-frame run (if we have one, build it if not) and put up a one-time warning window if its not found